src/main/mondrian/rolap/RolapCubeMember.java

```diff
@@ -201,6 +200,10 @@ public Object getPropertyValue(String propertyName, boolean matchCase) {
         return member.getPropertyValue(propertyName, matchCase);
     }
 
+    public Object getPropertyValue(String propertyName) {
```

Comment:
It is called from [here].(https://github.com/pentaho/mondrian/blob/master/src/main/mondrian/xmla/XmlaHandler.java#L2277)
There is an overridden [method](https://github.com/pentaho/mondrian/blob/master/src/main/mondrian/rolap/RolapCubeMember.java#L166) in RolapCubeMember, but it's overloaded version existed only in base class - DelegatingRolapMember.